mlir/lib/Dialect/Rock/Generator/ConvGenerator.cpp

```diff
@@ -835,9 +835,7 @@ LogicalResult ConvGenerator::genConvModule(ModuleOp &module, int rawKernelId,
   // Construct the FuncOp.
   func = func::FuncOp::create(builder.getUnknownLoc(), kernelName, funcType,
                               ArrayRef<NamedAttribute>(kernelAttrs));
-  // TODO[split-K]: split-K does not work with BwdWeight
-  if (!config.disableSplitKForTuning &&
-      config.operation.value() != ConvOpType::BwdWeight) {
```

Comment:
Why is this removed ? 

This PR is only adding parsing and conversion for bwd convolution. Doesn't seem to be handling splitK for the bwdWeight. I